commons/ava_networks/high_level_gecko_client.go

```diff
@@ -39,6 +39,10 @@ func NewHighLevelGeckoClient(
 	}
 }
 
+func (highLevelGeckoClient HighLevelGeckoClient) GetLowLevelClient() *gecko_client.GeckoClient {
```

Comment:
Rather than 1) creating a HighlevelClient and 2) then having a method to get the underlying lowlevel client, just use the lowlevel client that was passed in to create this?